docker-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Run the container the default way
+if [ "$1" = 'timesketch' ]; then
+	# Set SECRET_KEY in /etc/timesketch.conf if it isn't already set
+	if grep -q "SECRET_KEY = u''" /etc/timesketch.conf; then
+		OPENSSL_RAND=$( openssl rand -base64 32 )
+		# Using the pound sign as a delimiter to avoid problems with / being output from openssl
+		sed -i 's#SECRET_KEY = u\x27\x27#SECRET_KEY = u\x27'$OPENSSL_RAND'\x27#' /etc/timesketch.conf
+	fi
+
+	# Set up the Postgres connection
+	if [ $POSTGRES_USER ] && [ $POSTGRES_PASSWORD ] && [ $POSTGRES_ADDRESS ] && [ $POSTGRES_PORT ]; then
```

Comment:
Yup! Totally possible. As mentioned in the [Docker Hub documentation](https://hub.docker.com/r/jessemillar/timesketch/), you run the Timesketch container with a command similar to this:
```
docker run -d -e POSTGRES_USER=timesketch -e POSTGRES_PASSWORD=password -e TIMESKETCH_USER=admin -e TIMESKETCH_PASSWORD=password -e POSTGRES_ADDRESS=postgres -e POSTGRES_PORT=5432 -e ELASTIC_ADDRESS=elastic -e ELASTIC_PORT=9200 jessemillar/timesketch
```
Any of those values can be changed to link to any Postgres/Elastic install.

You could also use `docker-compose` and modify lines [10](https://github.com/jessemillar/timesketch/blob/492ea1f93ac18b00b2770280f271f501d495d426/docker-compose.yml#L10) and [31](https://github.com/jessemillar/timesketch/blob/492ea1f93ac18b00b2770280f271f501d495d426/docker-compose.yml#L31) to set up and use a Postgres server with a custom password. [Using `docker-compose` in production](https://docs.docker.com/compose/production/) is normal/supported.

For security, you'd most likely set a password environment variable on the machine that runs the `docker run` or `docker-compose` command and reference it in the `docker run` command or `docker-compose.yml` file a la `$SEKRET_POSTGRES_PASSWORD`.